M2/Macaulay2/packages/Macaulay2Doc/overview4.m2

```diff
@@ -5,7 +5,8 @@
 undocumented (hypertext, Hypertext)
 
 document {
-     Key => {hypertext,(hypertext, List),(hypertext, Sequence)},
+--     Key => {hypertext,(hypertext, List),(hypertext, Sequence)},
```

Comment:
Instead of commenting this line, could you remove it and add `undocumented methods html` at the top? This prevents lots of undocumented method warnings.

Reply:
do you mean `undocumented methods hypertext`? or `html`, or both?

Reply:
Sorry, I meant `hypertext`.